pkg/distributor/distributor.go

```diff
@@ -280,7 +280,7 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 			Namespace: "cortex",
 			Name:      "distributor_received_samples_total",
 			Help:      "The total number of received samples, excluding rejected, forwarded and deduped samples.",
-		}, []string{"user"}),
+		}, []string{"user", mimirpb.SampleMetricTypeLabel}),
```

Comment:
Why do we want to track these metrics per "sample type"?

Reply:
removed